mezzanine/accounts/__init__.py

```diff
@@ -75,6 +75,20 @@ def get_profile_form():
                                    settings.ACCOUNTS_PROFILE_FORM_CLASS)
 
 
+def get_profile_inline_form():
```

Comment:
I find the name of this function misleading. It currently implies that a form class is returned, not an admin class. I suggest we rename it to `get_profile_inline_admin()`.